lib/glific/third_party/gcs/gcs.ex

```diff
@@ -177,4 +179,72 @@ defmodule Glific.GCS do
       )
     end
   end
+
+  @spec bucket_name(non_neg_integer()) :: String.t() | nil
+  defp bucket_name(org_id) do
+    case get_secrets(org_id) do
+      %{"bucket" => bucket} -> bucket
+      _ -> nil
+    end
+  end
+
+  @doc """
+  Enabling bucket logging for the specified organization
+  """
+  @spec enable_bucket_logs(non_neg_integer()) :: {:ok, String.t()} | {:error, String.t()}
+  def enable_bucket_logs(org_id) do
```

Comment:
we should also call this function when setting credentials and also run this for existing organizations
similar to `Glific.Seeds.SeedsMigration.submit_otp_template_for_org(org_id)`